doc/NewcomerGuide.md

```diff
@@ -0,0 +1,44 @@
+# Newcomer Guide
+
+## Getting started
+Please read this document carefully before you start development on PowerPointLabs. You can always come back and refer when you have doubts/forget about anything.
+
+This document would assume that you have already forked the PowerPointLabs and PowerPointLabs-Website repositories on your local computer.
+
+For official Microsoft documentation on PowerPoint API, visit the website [here](https://msdn.microsoft.com/en-us/library/microsoft.office.interop.powerpoint(v=office.14).aspx).
+
+## Workflow of PowerPointLabs
+1. Create a branch from `PowerPointLabs/dev-release`. Your branch name should follow this convention: `issueNumber-description-of-issue` e.g. `123-fix-blur-incorrect-selection`
+1. Fix bugs or develop on that branch
+1. Do a pull request to `PowerPointLabs/dev-release` from your branch. The pull request title should be same as the issue title with the issue number included (refer to past pull requests). Give a brief description of what you fixed and the reasoning behind your fix to aid the reviewer
+1. Most of your codes will be in dev-release until they are properly tested. When ready, they will be merged into master and released to the public.
+
+## Workflow of PowerPointLabs-Website
+1. Create a branch from `master`
+1. Make desired changes to website and test locally
+1. Make a pull request with similar naming convention as above
+1. Once the change is accepted, wait for reviewer to merge the branch into `master` and upload the website for the public. Ensure that the [website](http://www.comp.nus.edu.sg/~pptlabs/) can be viewed and displays the latest changes correctly
+
+## Creating a New Lab
+As a rule of thumb:
+1. Your codes should never use `#pragma warning disable 0618`. Use methods from ActionFramework instead
+1. All features/bug fixes should be accompanied by tests and documentation
+1. When creating a new lab, use the following existing labs as a guide:
+   1. If lab does not require a pane: *Crop Lab*
+   1. If lab requires a pane: *Sync Lab*
+1. Note that any new labs would require accompanying Unit/Functional Tests. Use existing tests as reference
```

Comment:
Recommended to use Unit Tests instead of functional test because FT is much slower than UTs. UTs also open a powerpoint for testing in the background.